netlify.toml

```diff
@@ -1,2 +1,3 @@
 [build]
-  ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF -- docs"
+  ignore = "if  [ ${COMMIT_REF} == ${CACHED_COMMIT_REF} ] ; then false ; else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF -- docs ; fi"
```

Comment:
It's unfortunate that Netlify [chose ignore == 1 to mean "do not ignore."](https://docs.netlify.com/configure-builds/ignore-builds/#custom-ignore-command)

Reply:
Agreed!